vars/sectionDeployToAKS.groovy

```diff
@@ -81,9 +85,11 @@ def call(params) {
           onFunctionalTestEnvironment(environment) {
             stage("Functional Test - AKS ${environment}") {
               testEnv(aksUrl) {
-                pcr.callAround("functionalTest:${environment}") {
-                  timeoutWithMsg(time: 40, unit: 'MINUTES', action: 'Functional Test - AKS') {
-                    builder.functionalTest()
+                pcr.callAround("functionalTest-aks:${environment}") {
```

Comment:
why do you need this?

Reply:
We need to override environment targets and test URLs for AKS but not for ASE. https://github.com/hmcts/idam-api/blob/preview/Jenkinsfile_CNP#L55:L63

Reply:
but can't you do `smoketest:idam-preview` ? how is it special on AKS and not ASE?

Reply:
It's a while ago now but I think the reason I put this in place if because with the addition of AKS deployments for PRs and removal of spinInfra, we lost the environment vars that were being set from the terraform output. This meant we needed to set them and so I set them in the callbacks but the variable is different for ASE and AKS so I couldn't use `smoketest:idam-preview` because that set it for both.

Reply:
but you're only setting this?
https://github.com/hmcts/idam-api/blob/preview/Jenkinsfile_CNP#L58

Reply:
Yes and also overriding the integration environment with https://github.com/hmcts/idam-api/blob/preview/Jenkinsfile_CNP#L57. We also do similar things in web-public and web-admin. We would not want to set this globally or for ASE as it will break ASE smoke and functional tests.

Reply:
@timja we are planning to be fully over to AKS within the next two months, and at that time we can revert these changes. With the custom branches being blocked we will have no choice but to do this for now

Reply:
This isn't a blocker, just trying to understand, I don't fully understand why its needed, but dw it doesn't cause any harm